frontend/packages/topology/integration-tests/features/topology/topology-filter-bar-display.feature

```diff
@@ -5,59 +5,37 @@ Feature: Workload Groupings in Topology
 
         Background:
             Given user is at developer perspective
+              And user has created or selected namespace "aut-topology-display-options"
+              And user is at Add page
 
 
-        @regression @to-do
+        @smoke
         Scenario: Default state of Display dropdown: T-16-TC01
-            Given user is at the Topology page
+            Given user has created workload "nodejs-ex-git" with resource type "Deployment"
```

Comment:
```suggestion
            Given user has created workload "nodejs-ex-git" with resource type "Deployment"
            And user is at Topology page
```

Reply:
@makambalaji Post creating a workload, it will automatically be redirected to the topology page, I don't think, it's needed.

Reply:
Yeah true
IMO, this step gives more clarity in manual testing view. It's upto you, whether to add the step or not

Reply:
I guess it's not required as it's pretty much clear what we are testing and where it is located. I will resolve this conversation and update the PR.

Reply:
Ok